### DISCUSSION TITLE
Auto download Vercel url

### DISCUSSION THREAD

**Discussion by BibhabenduMukherjee** [vercel/community#1450] [Help] [answered]
### Summary

I create a next js app when I try to click to the url (Vercel  url) it's download something instead of opening the page.some Time the page opes sometime not. Please help me out

**Comment by swarnava** [comment, marked as answer]
Hi,

Do you have a public repository of the code which we can look into?

**Reply by BibhabenduMukherjee** [reply]
yes https://github.com/BibhabenduMukherjee/Blogging

**Reply by swarnava** [reply]
I couldn't build the project locally with following error:

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/1162991/217029608-9efb0ad1-c9c5-4cb1-8129-997157d4174d.png">

Also I will recommend trying our latest Next.js version

**Comment by BibhabenduMukherjee** [comment]
Dear sir , The error happenes just for the Sanity api id . I have it in my
env file.

**Reply by swarnava** [reply]
Can you provide a dummy key which we can check?

**Comment by BibhabenduMukherjee** [comment]
you can use all the env bellow maintained (but sanity project Id would not
going to be work it's a dummy value if you request then i can provide you
the original one )

NEXT_PUBLIC_GITHUB_ID=12gx19442b2113873112
NEXT_PUBLIC_GITHUB_SECRET=771e98v12046e404564fe8bb2da8f46347b5886

NEXT_PUBLIC_SANITY_PROJECT_ID=32t7656d
NEXT_PUBLIC_SANITY_DATASET=production
NEXT_PUBLIC_SANITY_API_VERSION=2022-11-15

NEXTAUTH_SECRET=my_secret
NEXT_PUBLIC_FIREBASE_APIKEY=AjhdCIT6IQ3cJC4Cuf-7ui8fOBnYCvloXsPw6
NEXT_PUBLIC_FIREBASE_AUTHDOMAIN=codlify-87h764.firebaseapp.com
NEXT_PUBLIC_FIREBASE_DATABASEURL=
https://codlify-87h764-default-rtdb.asia-southeast1.firebasedatabase.app
NEXT_PUBLIC_FIREBASE_PROJECTID=codlify-87h764
NEXT_PUBLIC_FIREBASE_STORAGEID=codlify-87h764.appspot.com
NEXT_PUBLIC_FIREBASE_MESSID=331193114928
NEXT_PUBLIC_FIREBASE_APPID=1:931803107924:874831a96b313c1

Sir please check this production edition on this link*
https://codelify.netlify.app
<https://codelify.netlify.app> (click any post and refresh the page you can
see the problem)*
I was facing a problem with vercel then i hosted it to netlify my problem
reduced by 50%. but in vercel i cannot do

**Comment by swarnava** [comment]
I tried your env variable and still received error

<img width="1180" alt="Screenshot 2023-02-12 at 9 55 35 AM" src="https://user-images.githubusercontent.com/1162991/218292946-e7e2a467-0819-4b3f-8e38-f8fa6035935f.png">

**Comment by BibhabenduMukherjee** [comment]
Thank you for your time i have  solved the issue.It's happens for wrong app directory configuration
